src/sider/oppfolgingsplan/oppfolgingsplaner/BeOmOppfolgingsplan.tsx

```diff
@@ -71,46 +86,44 @@ export default function BeOmOppfolgingsplan({
     });
   }
 
-  const lastForesporselCreatedAt =
-    getOppfolgingsplanForesporsel.data?.[0]?.createdAt;
-
-  const isAktivForesporsel = !!lastForesporselCreatedAt
-    ? currentOppfolgingstilfelle.start <= lastForesporselCreatedAt &&
-      lastForesporselCreatedAt <= currentOppfolgingstilfelle.end
-    : false;
-  const aktivForesporselTekst = `${
-    texts.aktivForesporsel
-  } ${tilLesbarDatoMedArUtenManedNavn(lastForesporselCreatedAt)}`;
-
   return (
     <>
-      {postOppfolgingsplanForesporsel.isSuccess && (
-        <Alert variant="success" className="mb-2" size="small">
-          Forespørsel om oppfølgingsplan ble sendt
+      {isAktivForesporsel && (
+        <Alert variant="warning" className="mb-2">
+          {aktivForesporselTekst}
         </Alert>
       )}
       <Box background="surface-default" className="mb-4 flex flex-col p-4">
-        <Heading size="medium">{texts.header}</Heading>
+        <Heading size="small" level="3">
+          {texts.header}
+        </Heading>
         <BodyLong className="mb-2">{texts.description}</BodyLong>
         <BodyShort>
           {texts.virksomhet} {aktivNarmesteLeder.virksomhetsnavn}
         </BodyShort>
         <BodyShort className="mb-2">
           {texts.narmesteLeder} {aktivNarmesteLeder.narmesteLederNavn}
         </BodyShort>
-        {isAktivForesporsel && !!lastForesporselCreatedAt && (
-          <Alert inline variant="warning" className="mb-2">
-            {aktivForesporselTekst}
+        {!postOppfolgingsplanForesporsel.isSuccess && (
+          <Button
+            className="w-fit mb-2"
+            size="small"
+            onClick={onClick}
+            loading={postOppfolgingsplanForesporsel.isPending}
+          >
+            {texts.button}
+          </Button>
```

Comment:
Så knappen skal ikke være mulig å klikke på hvis man allerede har sendt en forespørsel (med mindre man refresher)?

Reply:
Man ønsker å vise en Alert i stedet for knappen rett etter man har sendt en foresporsel? Kunne tydeliggjort det i testene kanskje. Og evt forenklet til ternary `postOppfolgingsplanForesporsel.isSuccess ? <Alert> : <Button> ` :thinking: 
Og når man går inn på siden igjen så skal man kunne gjøre ny foresporsel selv om det finnes en aktiv?

Reply:
Stemmer!

Reply:
Av en eller annen grunn så jeg bare Eirik sin kommentar når jeg svarte 😅 Lurt med ternary! Legger det på. Kan også prøve å tydeliggjøre det i testene.
Det siste stemmer. Man skal kunne gjøre en ny forespørsel, selv om det finnes en aktiv 😊